WalletWasabi.Documentation/HowToDeploy.md

```diff
@@ -0,0 +1,25 @@
+# The agreed procedure to deploy something to production
+
+1. Create a summary of what is going to be deployed (we are still working on this and don't have anything automatic yet).
```

Comment:
For the last deployment to testnet I used a label `affiliate` so Satoshi Labs and Kukks can see what is are the changes that they could be more interested on. This is not perfect but let see how it works and in case it is not enough we can try something else or more elaborated.

Reply:
If they are OK with that it is perfect for me. Can you add that to the document?